src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1988,11 +2008,12 @@ PreserveConstDictMatMulSymm::PreserveConstDictMatMulSymm(PreserveConstDictMatMul
 
         if ((ov::element::f8e4m3 == matched_qweight->get_element_type() ||
              ov::element::f8e5m2 == matched_qweight->get_element_type() ||
-             ov::element::f8e8m0 == matched_qweight->get_element_type()) &&
+             ov::element::f8e8m0 == matched_qweight->get_element_type() ||
+             ov::element::i8 == matched_qweight->get_element_type()) &&
```

Comment:
It is better to rename this whole pattern from `PreserveConstDictMatMulSymm` to `PreserveConstDictMatMulFP8` and not to handle `i8` here at all - there's no need in this